runtime/shared/src/sys/linux/renderstruct.cpp

```diff
@@ -23,6 +23,7 @@ void r_InitRenderStruct(bool bFullClear)
 {
     RenderStructInit rsi{0,bFullClear, '\0'};
     g_Render.Init(&rsi);
```

Comment:
problem is above oglrdr isn't initialized 
should likely be a static for this translation unit

Reply:
RenderStruct::g_Render I think is the one which is null (on linux)